ServerManager/api/editGameSession.php

```diff
@@ -25,7 +27,7 @@
     "upgrade", // called in JS function callUpgrade
     "changeGameState", // called in JS functions startSession, pauseSession
     "processZip", // called by server API gamesession/ArchiveGameSessionInternal
-    "recreate", // called by JS function RecreateSession
+    //"recreate", // called by JS function RecreateSession
```

Comment:
maybe we can just remove it
```suggestion
    //"recreate", // called by JS function RecreateSession
```